ios/MusicControlManager.m

```diff
@@ -85,23 +85,28 @@ - (dispatch_queue_t)methodQueue
 
         [details setValue:speed forKey:MEDIA_SPEED];
     }
-    if ([state isEqual:MEDIA_STATE_STOPPED]) {
-        MPRemoteCommandCenter *remoteCenter = [MPRemoteCommandCenter sharedCommandCenter];
-        [self toggleHandler:remoteCenter.stopCommand withSelector:@selector(onStop:) enabled:false];
-    }
 
+    if ([state isEqual:MEDIA_STATE_STOPPED]) {
+         MPRemoteCommandCenter *remoteCenter = [MPRemoteCommandCenter sharedCommandCenter];
+         [self toggleHandler:remoteCenter.stopCommand withSelector:@selector(onStop:) enabled:false];
+     }
+    
+    
     NSMutableDictionary *mediaDict = [[NSMutableDictionary alloc] initWithDictionary: center.nowPlayingInfo];
 
     center.nowPlayingInfo = [self update:mediaDict with:details andSetDefaults:false];
-
-    // Playback state is separated in 11+
-    if (@available(iOS 11.0, *)) {
-        if ([state isEqual:MEDIA_STATE_PLAYING]) {
-            center.playbackState = MPNowPlayingPlaybackStatePlaying;
-        } else if ([state isEqual:MEDIA_STATE_PAUSED]) {
-            center.playbackState = MPNowPlayingPlaybackStatePaused;
-        } else if ([state isEqual:MEDIA_STATE_STOPPED]) {
-            center.playbackState = MPNowPlayingPlaybackStateStopped;
+    
+    // tvos do not support playbackstate proprety
+    if(TARGET_OS_IOS) {
```

Comment:
Is this needed? I don't think it is since the available check below looks specifically for iOS anyway

Reply:
yep we need this